### ISSUE TITLE
How do I delete a tensorrt profile? If I delete the file it continues to show up in the webgui, and throws errors, because it realizes a profile is missing.

### ISSUE THREAD

**Issue by left1000** [AUTOMATIC1111/stable-diffusion-webui-tensorrt#73]
How do I delete a tensorrt profile? If I delete the file it continues to show up in the webgui, and throws errors, because it realizes a profile is missing.

**Comment by left1000** [comment]
Right now, the inability to easily delete one profile without deleting all the profiles for given checkpoint, is making it hard to test things. And leading to me making some likely false statements over on https://github.com/AUTOMATIC1111/stable-diffusion-webui-tensorrt/issues/69

I think maybe the extension needs a delete profile button in the webgui.

**Comment by left1000** [comment]
Force rebuild results in the profile being listed in the list of profiles twice so even force rebuild doesn't serve as an effective delete profile option like I had hoped it might.

**Comment by japtain-cack** [comment]
So it must read the `model.json` in the `Unet-trt` directory. I shut down the server, deleted the file from the `Unet-trt` and `Unet-onnx` directories, then removed the json entries from the `model.json` file. After starting a1111 again, the entry was gone.

**Comment by XPHUPZHJ** [comment]
Why not just delete the whole content of `Unet-trt` e.g. also deleting the `model.json`-file without trying to make changes to it?

**Comment by left1000** [comment]
Yep, that's right, deleting every single file in there will fix the "problem" and waste 10-15minutes of my life. Which is how I explained it in the first place.....
